server/sql_app/main.py

```diff
@@ -155,8 +156,13 @@ def create_study_area(
     db_session = crud.get_session(db, session_id=session_id)
     if db_session is None:
         raise HTTPException(status_code=404, detail="Session not found")
-    return crud.create_study_area(
+    db_study_area = crud.create_study_area(
         db=db, **new_area.dict(), session_id=session_id)
+    # scenario = schemas.ScenarioBase
+    # scenario.name = 'baseline'
+    # _ = crud.create_scenario(
+    #     db=db, scenario=scenario, study_area_id=db_study_area.id)
```

Comment:
Should this be commented out? Is the frontend handling the baseline creation?

Reply:
Thanks, I had an unpushed commit that remove removed some commented code like this. Just pushed.